src/legal-officer/LocRequestsHistory.tsx

```diff
@@ -24,6 +26,16 @@ export default function LocRequestsHistory() {
                     render: request => <Cell content={ request.description } />,
                     align: "left",
                 },
+                {
```

Comment:
Is it intentional that the reject reason is not shown ? That's also the case for both protection and tokenization requests. To be addressed with Product Owner ?!

Reply:
Indeed, I'll ask.